packages/backend/prisma/schema.prisma

```diff
@@ -11,12 +11,18 @@ datasource db {
 }
 
 model Link {
-  id    String @id @default(uuid())
-  href  String
+  id           String     @id @default(uuid())
+  href         String
+  collectionId String
+  collection   Collection @relation(fields: [collectionId], references: [id])
 }
 
 model Collection {
-  id    String @id @default(uuid())
+  id        String @id @default(uuid())
+  links     Link[]
```

Comment:
Will this populate automatically?

Reply:
I do not understand the question

Reply:
Whether Prisma will join the link table to the collection table when we query the Collection model. I.e. wil this list be empty or will it contain links, do we have to define a relation or something like that?

Reply:
It will be and empty list until somebody adds links, but links needs a collection for create

Reply:
It will be and empty list until somebody adds links, but links needs a collection for create